src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -732,6 +737,10 @@ export class CodeModelCliImpl implements CodeModelAz
         }
     }
 
+    public get Method_IsLongRun(): boolean
+    {
+        return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentMethodIndex].extensions['x-ms-long-running-operation']? true: false;
```

Comment:
>? true: false [](start = 156, length = 13)

you dont need this

Reply:
but what if the x-ms-long-running-operation value changes in the future.

Reply:
i mean you can return the boolean directly, no need to ? true : false

Reply:
I know. I mean what if the x-ms-long-running-operation value  changes to objects etc.

Reply:
is it expected for x-ms-long-running-operation to be object instead of bool?

Reply:
No, Just a thought.

Reply:
or do we want it be ignored or to report error if some one set that to object instead of bool?

Reply:
I just think that if we don't explicitly return a bool for some usage like `model.Is_LongRun ===  false`.  it will probably fail when x-ms-long-running-operation is not set.

Reply:
i see. that's a good point.